psychopy/plugins/__init__.py

```diff
@@ -830,7 +830,7 @@ def loadPlugin(plugin):
 
                 return False
 
-            if '.zip' in ep.__file__:
+            if hasattr(ep, 'file') and '.zip' in ep.__file__:
```

Comment:
Shouldn't it be `hasattr(ep, '__file__')`?

Reply:
but at least in my case it meant the crash didn't occur 😉